File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int getHeight(void);
void printPyramid(int height);

int main(void)
{
    // get pyramid height
    int height = getHeight();


    printPyramid(height);

}

int getHeight(void)
{
    // get height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    return height;
}

void printPyramid(int height)
{
    // loop height
    for (int i = 1; i <= height; i++)
    {
        // print gaps
        for (int j = 1; j <= height - i; j++)
        {
            printf(" ");
        }

        // print first pyramid
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }

        // separate pyramids
        printf("  ");

        // print second pyramid
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }

        // separate height levels
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    height = get_height()

    for i in range(height):
        # print blanks
        print(" " * (height - 1 - i), end="")

        # left side
        print("#" * (i + 1), end="")

        # separate pyramids
        print("  ", end="")

        # right side and newline
        print("#" * (i + 1))


def get_height():
    # get height in range 1-8
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            return height


main()